File: poo2/projeto/php/modelo/bobina.php
<?php
    include_once 'produto.php';
    include 'fabricante.php';
    class Bobina extends Produto 
    {
        
        public $qte_polos;

        function Bobina($numero_peca, $altura, $largura, $comprimento, $diamentroInterno, $diamentroExterno,$amperagem,$voltagem,$valor,$marca,$numero_oem,$qte_polos)
        {
            $this->numero_peca      = $numero_peca;
            $this->altura           = $altura;
            $this->largura          = $largura;
            $this->comprimento      = $comprimento;
            $this->diamentroInterno = $diamentroInterno;
            $this->diamentroExterno = $diamentroExterno;
            $this->amperagem        = $amperagem;
            $this->voltagem         = $voltagem;
            $this->valor            = $valor;
            $this->qte_polos        = $qte_polos;             
            $this->fabricante       = new Fabricante($marca, $numero_oem);
        }
    }
?>

File: projeto/ServeletAutopecas/src/java/LN/Bobina.java
package LN;
public class Bobina extends Produto{
    int qte_polos;

    public int getQte_polos() {
        return qte_polos;
    }

    public void setQte_polos(int qte_polos) {
        this.qte_polos = qte_polos;
    }
     
    public  Bobina(int numeroPeca, String altura, String largura, String comprimento, String diamentroInterno, String diamentroExterno, float valor, float amperagem, float voltagem, int qte_polos, String nome, String numeroOEM) {
        super.setNumeroPeca(numeroPeca);
        super.setAltura(altura);
        super.setLargura(largura);
        super.setComprimento(comprimento);
        super.setDiamentroExterno(diamentroExterno);
        super.setDiamentroInterno(diamentroInterno);
        super.setValor(valor);
        super.setAmperagem(amperagem);
        super.setVoltagem(voltagem);
        setQte_polos(qte_polos);
    }
   
}
